rest/route/service.go

```diff
@@ -203,6 +203,7 @@ func AttachHandler(app *gimlet.APIApp, opts HandlerOpts) {
 	app.AddRoute("/projects/{project_id}/task_reliability").Version(2).Get().Wrap(requireUser).RouteHandler(makeGetProjectTaskReliability(opts.URL))
 	app.AddRoute("/projects/{project_id}/task_stats").Version(2).Get().Wrap(requireUser, viewTasks).RouteHandler(makeGetProjectTaskStats(opts.URL))
 	app.AddRoute("/projects/{project_id}/versions").Version(2).Get().Wrap(requireUser, viewTasks).RouteHandler(makeGetProjectVersionsHandler(opts.URL))
+	app.AddRoute("/projects/{project_id}/versions").Version(2).Post().Wrap(requireUser, editTasks).RouteHandler(makeSetPriorityProjectVersionsHandler(opts.URL))
```

Comment:
The route we currently have that allows setting priority requires users to have `submitPatches` permissions.  I don't know if `editTasks` is enough here.

Reply:
Good point, changed it to `submitPatches`